getting-started/README.md

```diff
@@ -1,110 +1,108 @@
+```bash
+docker compose logs tgi-service | grep Connected
 ```
-#xeon
-cd docker_compose/intel/cpu/xeon/
-#gaudi
-cd docker_compose/intel/hpu/gaudi/
-#nvidia
-cd docker_compose/nvidia/gpu/
+The output shows `Connected` as shown:
 ```
-Now we can start the services
+tgi-service | 2024-10-18T22:41:18.973042Z INFO text_generation_router::server: router/src/server.rs:2311: Connected
 ```
-docker compose up -d
-```
-It will automatically download the needed docker images from docker hub:
 
-- docker pull opea/chatqna:latest
-- docker pull opea/chatqna-ui:latest
+### Interact with ChatQnA
 
-In the following cases, you will need to build the docker image from source by yourself.
+You can interact with ChatQnA via a browser interface:
+* Under `Infrastructure` in the left pane, go to `Network/Security groups/<Your Security Group>/Rules`
+* Select `Create`
+* Enable inbound traffic for port 80
+* Open a browser and navigate [http://{external_public_ip}:80](http://{external_public_ip}:80") to view the ChatQnA interface.
 
-- The docker image failed to download. (You may want to first check the
-  [Docker Images](https://github.com/opea-project/GenAIExamples/blob/main/docker_images_list.md)
-  list and verify that the docker image you're downloading exists on dockerhub.)
-- You want to use a different version than latest.
+For more information on editing inbound/outbound rules, click [here](https://cloud.ibm.com/docs/vpc?topic=vpc-updating-the-default-security-group&interface=ui)
 
-Refer to the {ref}`ChatQnA Example Deployment Options <chatqna-example-deployment>` section for building from source instructions matching your hardware.
+A snapshot of the interface looks as follows:
 
-## Interact with ChatQnA Megaservice and Microservice
-Before interact ChatQnA Service, make sure the TGI service is ready (which takes up to 2 minutes to start).
-```
-docker ps
-# expected: all images's status are up
-# TGI example on on Xeon and Nvidia GPU
-docker logs tgi-service | grep Connected
-# TGI example on on Gaudi
-docker logs tgi-gaudi-service | grep Connected
-# execpted output: ... INFO text_generation_router::server: router/src/server.rs:2311: Connected
-```
-```
+![Chat Interface](assets/chat_ui_response.png)
+
+
+> **Note:** this example leverages the Nike 2023 Annual report for its RAG based content. See the [ChatQnA Sample Guide](https://opea-project.github.io/latest/examples/ChatQnA/ChatQnA_Guide.html)
+to learn how you can customize the example with your own content. 
+
+To interact with the ChatQnA application via a `curl` command:
+
+```bash
 curl http://${host_ip}:8888/v1/chatqna \
     -H "Content-Type: application/json" \
     -d '{
         "messages": "What is the revenue of Nike in 2023?"
     }'
 ```
-This command will provide the response as a stream of text. You can modify the `message` parameter in the `curl` command and interact with the ChatQnA service.
+ChatQnA provides the answer to your query as a text stream.
+

```

Comment:
Do add a step to upload content to RAG system

Use the curl command below to upload a file.

curl -X POST "http://${host_ip}:6007/v1/dataprep" \
     -H "Content-Type: multipart/form-data" \
     -F "files=@./nke-10k-2023.pdf"

Reply:
@mkbhanda Can we add this in the next iteration? I'd have to validate it on IBM cloud

Reply:
sure